### PR TITLE
fix: modify declaration slice and map code style

### PR DESCRIPTION
**Pull request by demoManito**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
[golang codereview comments](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)
- empty slice should be `var i []int`, instead of `i := []int{}` 
- make map should be `make(map, size)`

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/versiongetter.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func (g *KubeVersionGetter) KubeletVersions() (map[string]uint16, error) {
 
 // computeKubeletVersions returns a string-int map that describes how many nodes are of a specific version
 func computeKubeletVersions(nodes []v1.Node) map[string]uint16 {
-	kubeletVersions := map[string]uint16{}
+	kubeletVersions := make(map[string]uint16, len(nodes))
```
[the remark this answers is not in the document]
Comment (demoManito) [reply cmd/kubeadm/app/phases/upgrade/versiongetter.go:108]: If there are no more than three, you really don't need to pre allocate the size. I'll modify it

[the remark this answers is not in the document]
Comment (demoManito) [reply cmd/kubeadm/app/phases/upgrade/versiongetter.go]: Thank you, it has been revised

**Comment by neolit123** [comment]
please keep the commits squashed to 1.

/ok-to-test

**Comment by demoManito** [comment]
@neolit123 CI has been successful. Please review it again

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**